File: CJY/BJ/No_1065.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout});

rl.on("line", (s) => {
    const n = parseInt(s);
    let cnt = n;
    if (n >= 100) {
        cnt = 99;
        for (let i = 100; i <= n; i++) {
            if (i === 1000) break;
            if (~~(i % 10) - ~~((i / 100) % 10) === (~~(i % 10) - ~~((i / 10) % 10)) * 2) cnt++;
      }
    }
    console.log(cnt);
    rl.close();
}).on("close", () => {process.exit();});

File: CJY/BJ/No_1065.java
import java.io.*;

public class No_1065 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        br.close();

        int cnt;
        if(n < 100) {
            cnt = n;
        } else {
            cnt = 99;
            for(int i = 100; i <= n; i++) {
                if (i == 1000) break;
                if((i % 10) - ((i / 100) % 10) == ((i % 10) - ((i / 10) % 10)) * 2) cnt++;
            }
        }

        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        bw.write(String.valueOf(cnt));
        bw.flush();
        bw.close();
    }
}
